aws/aws_kms.go

```diff
@@ -155,16 +158,22 @@ func (a *awsKmsSecrets) GetSecret(
 	if len(cipherBlob) == 0 {
 		cipherBlob = []byte(secretId)
 	}
+	// AWS KMS api requires the cipherBlob to be in base64 decoded format.
+	// Check if it is encoded and decode if required.
+	decodedCipherBlob, err = base64.StdEncoding.DecodeString(string(cipherBlob))
```

Comment:
now the generated data key will be base64 encoded by default or we need to encode the cipher text in put secrets as well?

Reply:
it is encoded by default.